README.md

```diff
@@ -1,5 +1,206 @@
+
+The following components make up our software stack:
+
+- Onboard:
+    * [Controls](onboard/catkin_ws/src/controls) - Determines thruster outputs given a current and desired state.
+    * [Computer Vision](onboard/catkin_ws/src/cv) - Locates objects (goals/obstacles) via camera input and machine learning.
+    * [Data Pub](onboard/catkin_ws/src/data_pub) - Collects and parses data from sensors and publishes it for use by other packages.
+    * [Execute](onboard/catkin_ws/src/execute) - Houses launch files that simplify starting and stopping our stack.
+    * [Motion Planning](onboard/catkin_ws/src/motion_planning)/[Task Planning](onboard/catkin_ws/src/task_planning) [WIP] - Determines and defines the tasks and motion of the robot.
+    * [Offboard Comms](onboard/catkin_ws/src/offboard_comms) - Allows communication between ROS code and the onboard Arduino.
+    * [RoboSub Description](onboard/catkin_ws/src/robosub_description) - Contains files to describe and display our robot.
+    * [Sensor Fusion](onboard/catkin_ws/src/sensor_fusion) - Interprets sensor data and publishes an estimation of the current robot state.
+    * [Static Transforms](onboard/catkin_ws/src/static_transforms) - Makes static transforms available for other packages.
+- Landside:
+    * [Camera View](landside/catkin_ws/src/camera_view) - Package that allows for viewing, saving, and loading videos to simulate camera input.
+    * [Joystick](landside/catkin_ws/src/joystick) - Allows manual joystick control for testing.
+    * [Simulation](simulation) - Physics-enabled simulation that can be used for local testing. (not in landside, but run using the landside container)
+
+
+## Flow 
+
+The general flow of information between components is shown in the diagram below:
+
+(Diagram to be added)
+
+
+## Running Our Docker Images
+
+### Required Software
+1. Download and install the appropriate Docker client.
+
+    * [Mac](https://docs.docker.com/docker-for-mac/install/)
+    * [Windows (Home)](https://docs.docker.com/docker-for-windows/install-windows-home/)
+    * [Windows (Pro, Education, etc.)](https://docs.docker.com/docker-for-windows/install/)<br>
+        :information_source: Which Windows do I have? *Settings > System > About > look at "Edition"*
+
+1. If you want graphics forwarding over SSH:
+
+    * [Mac: XQuartz](https://www.xquartz.org/)
+    * [Windows: MobaXterm](https://mobaxterm.mobatek.net/)<br>
+        :warning: Open MobaXterm > click on Settings > go the X11 tab > uncheck RANDR. This allows us to use [rviz](http://wiki.ros.org/rviz) over SSH.
+
+### Pool Testing
+Use these instructions when running code on the robot itself.
+
+1. Connect your computer to the robot via Ethernet. Then SSH into the robot's onboard computer.
+    ```bash
+    ssh robot@192.168.1.1
+    ```
+1. There, make sure the latest code from this repo is pulled. Then, run the onboard container if it's not already running.
```

Comment:
numbering for this and following steps

Reply:
see previous comment on numbering. tl;dr we'll keep it as is.